### DISCUSSION TITLE
Link Button Dropdown

### DISCUSSION THREAD

**Discussion by eleifer-twilio** [twilio-labs/paste#1848] [Ideas] [answered]
Hi Team, 

I would like to create a dropdown but want the hierarchy to be tertiary. Can I create a dropdown off of a link using paste?

Please advise.

![Screen Shot 2021-09-08 at 5 05 00 PM](https://user-images.githubusercontent.com/73968607/132601721-8452b331-1d57-4945-bcf2-359ee53e005b.png)

Thank you!

**Comment by kate-porter** [comment, marked as answer]
Hey @eleifer-twilio - thanks for asking about this. You won't be able to use the actual Anchor component for this, as that component can only support linking to a page. However, the [Link-style Button](https://paste.twilio.design/components/button#examples) should work for you here. Let us know if that makes sense or if you have any other questions!

**Reply by TheSisb** [reply]
Specifically [this Menu example](https://paste.twilio.design/components/menu#choosing-a-menu-trigger) but change variant to `link`

**Reply by eleifer-twilio** [reply]
Hi @katieporter @TheSisb Sorry I did not mean anchor but the link button. I don't see an example of a menu off of a link button.  Is this something we can do? Do we need to add a carrot on the link?

![Screen Shot 2021-09-09 at 8 05 44 AM](https://user-images.githubusercontent.com/73968607/132711807-ea989ee9-fdfe-4496-80f2-0aa7f0bc2066.png)

**Reply by kate-porter** [reply]
Yep, like Shadi mentioned, you can open a menu using any variant of Button, including a Link-style Button. Including the chevron on the right would be ideal so it's clear to the user that it's going to open a menu.

**Reply by eleifer-twilio** [reply]
Perfect! Thank you @katieporter

**Reply by kate-porter** [reply]
Sure thing! There's a Link-style variant for the Button component in Figma that you can use. :)
